### PR TITLE
For 0 5 0 release

### PR DESCRIPTION
**Pull request by DavidGoldwasser**

### CODE REVIEW

**Comment by DavidGoldwasser** [comment]
@mdahlhausen I think I need to update the min version for nze_hvac measure to 3.3 to avoid this error when CI runs on older installer `undefined method `remove_hvac' `

**Comment by mdahlhausen** [comment]
yeah, that makes sense

File: openstudio-ee.gemspec [hunk 1]
```diff
@@ -28,8 +28,8 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = '~> 2.7.0'
 
   spec.add_dependency 'bundler', '~> 2.1'
-  spec.add_dependency 'openstudio-extension', '~> 0.4.3'
-  spec.add_dependency 'openstudio-standards', '~> 0.2.14'
+  spec.add_dependency 'openstudio-extension', '~> 0.4.4'
+  spec.add_dependency 'openstudio-standards', '~> 0.2.15.pre.rc1'
```
Comment (nllong) [inline openstudio-ee.gemspec:32]: Are we sure we want to release an actual version of this that points to a `pre.rc1` release of standards?

Comment (DavidGoldwasser) [reply openstudio-ee.gemspec:32]: @nllong Thanks for the catch, I'll update both gems here

Comment (DavidGoldwasser) [reply openstudio-ee.gemspec]: @nllong hold off something is wrong here. The change log and version object are also outdated, I thought I based this branch off of the testing branch. I'll make more more commit to fix things

**Review by nllong — APPROVED** [review]
great!

**Pull request merged**